restapi/src/main/java/io/stargate/web/docsapi/resources/DocumentResourceV2.java

```diff
@@ -136,6 +143,12 @@ public Response postDoc(
                   .getHeaderString(HttpHeaders.CONTENT_TYPE)
                   .toLowerCase()
                   .contains("application/json");
+
+          ExecutionContext context = ExecutionContext.NOOP_CONTEXT;
+          if (profile != null && profile) {
```

Comment:
nit: `Boolean.TRUE.equals(profile)` so much more readable :)

Reply:
thx, indeed!

Reply:
Actually, I moved this into a utility method on `ExecutionContext` in f2a5a8e6fa283c5db2ffe5f5b3c94ebdfccf9eb2